specs/relay-v1.rst

```diff
@@ -0,0 +1,410 @@
+.. _relay-v1:
+
+Relay Protocol v1
+=================
+
+What is a relay?
+----------------
+
+Relay is a service which relays data between two *devices* which are not able to
+connect to each other directly otherwise. This is usually due to both devices
+being behind a NAT and neither sides not being able to open a port which would

```

Comment:
2 negatives are not needed here.
"and neither side being able to open a port..."
would be fine.